core/server/data/migrations/init/2-create-fixtures.js

```diff
@@ -10,6 +11,15 @@ module.exports = function insertFixtures(options) {
 
     return Promise.mapSeries(fixtures.models, function (model) {
         logging.info('Model: ' + model.name);
+
+        // The Post model fixtures need a `published_at` date, where at least the seconds
+        // are different, otherwise `prev_post` and `next_post` helpers won't workd with
+        // them.
+        if (model.name === 'Post') {
+            _.forEach(model.entries, function (post, index) {
+                post.published_at = moment().add(index, 'seconds');
```

Comment:
It would be a bit odd having something in the model layer that's only relevant to fixtures? I don't think the model layer should be randomly adding time to the `published_at` field but I could be wrong.

My initial thoughts were to manually specify the `published_at` date strings in the `fixtures.json` file but this seemed to work just as well.